Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -1013,6 +1032,41 @@ void DrawSettings()
             }
         }
 
+        void Save()
+        {
+            var path = EditorUtility.SaveFilePanel("Save report to json file", s_SaveLoadDirectory, string.Format("project-auditor-report.json"), "json");
+            if (path.Length != 0)
+            {
+                m_ProjectReport.Save(path);
+
+                EditorUtility.RevealInFinder(path);
+
+                s_SaveLoadDirectory = Path.GetDirectoryName(path);
+
+                ProjectAuditorAnalytics.SendUIButtonEvent(ProjectAuditorAnalytics.UIButton.Save, ProjectAuditorAnalytics.BeginAnalytic());
+            }
+        }
+
+        void Load()
+        {
+            var path = EditorUtility.OpenFilePanel("Load from json file", s_SaveLoadDirectory, "json");
+            if (path.Length != 0)
+            {
+                m_LoadButtonAnalytic =  ProjectAuditorAnalytics.BeginAnalytic();
+
+                m_ProjectReport = ProjectReport.Load(path);
+                m_AnalysisState = AnalysisState.Valid;
+
+                s_SaveLoadDirectory = Path.GetDirectoryName(path);
+            }
+            OnEnable();
+
+            // update list of assembly names
+            var scriptIssues = m_ProjectReport.GetIssues(IssueCategory.Code);
+            m_AssemblyNames = scriptIssues.Select(i => i.GetCustomProperty((int)CodeProperty.Assembly)).Distinct().OrderBy(str => str).ToArray();
+            UpdateAssemblySelection();
```

Comment:
Good catch. That was me being lazy because I know I will need to refactor quite a bit of UI code in the future.

For the time being, I extracted that code to a _UpdateAssemblyNames()_ method.